<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPerson</class>
 <widget class="QDialog" name="editPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="nameText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genderText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Gender: </string>
   </property>
  </widget>
  <widget class="QLabel" name="dobText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>DOB:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dodText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>DOD:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="dobEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="dodEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="editText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>5</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Edit person</string>
   </property>
  </widget>
  <widget class="QLabel" name="deceasedText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string>Deceased: </string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>199</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="aliveButton">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="deadButton">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>199</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="maleButton">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="femaleButton">
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>maleButton</tabstop>
  <tabstop>femaleButton</tabstop>
  <tabstop>dobEdit</tabstop>
  <tabstop>aliveButton</tabstop>
  <tabstop>deadButton</tabstop>
  <tabstop>dodEdit</tabstop>
  <tabstop>submitButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>listView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
